lib/src/main/java/graphql/nadel/engine/NadelIncrementalResultSupport.kt

```diff
@@ -145,21 +143,6 @@ class NadelIncrementalResultSupport internal constructor(
         initialCompletionLock.complete(Unit)
     }
 
-    fun close() {
-        coroutineScope.cancel()
-    }
```

Comment:
HUH, this needs to be called. I think it's a bug that it's not.

i.e. we should cancel the jobs launched on the defer scope if the incremental result is cancelled e.g. on request closed early.

We don't have to address that in this PR.

I _suspect_ we can edit `resultFlow` to something like

```kotlin
    fun resultFlow(): Flow<DelayedIncrementalPartialResult> {
        return resultFlow
            .onCompletion {
                close()
            }
    }
```

Where `onCompletion` is invoked on cancellation.